### PR TITLE
[FEAT] : Note UPDATE Editor functionality added  

### PR DESCRIPTION
**Pull request by t007rushi**
Can open an editor on an already created note by 
clicking the edit icon
pops up an editor for note updation

recording

<details><summary>Browser metadata</summary>

```
Path:      /notes
Browser:   Chrome 99.0.4844.84 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--monkeep.netlify.app%2Fnotes&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-11--monkeep.netlify.app/notes) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=624cb804f7d4e50009fc881f&commentID=624cbac443622719c74ec8f9&resolution=resolved)

### CODE REVIEW

**Review by shaksham08 — APPROVED** [review]
Code looks well structured and clean. Its nice to see the use of matchPath from react router dom

File: src/components/NoteCard/NoteCard.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { useNotes } from "../../context/notes-context";
```
Comment (vijaytembugade) [inline src/components/NoteCard/NoteCard.jsx:2]: You can barrel import/export  the context.

**Review by vijaytembugade — APPROVED** [review]
Editing functionality working as smooth. Good to go.

**Review by dikshadhote — APPROVED** [review]
The code is clean and understantable.
Good work !

**Pull request merged**